ui/src/components/config_section/ActivityConfigSection.scss

```diff
@@ -1,11 +1,6 @@
 .activityConfigPanel {
   .euiDescriptionList {
-    .euiDescriptionList__title {
-      width: 110px;
-    }
-
-    .euiDescriptionList__description {
-      width: calc(100% - 110px);
-    }
+    column-gap: 0px !important;
+    row-gap: 0px !important;
```

Comment:
Similar to what was done for Turing in https://github.com/caraml-dev/turing/pull/384/files#r1677552781, these new styles because the description list component now has gaps by default between each cell.

The other existing styles were removed because they are now converted to a values of a `columnWidths` prop, just as it was done for Turing in https://github.com/caraml-dev/turing/pull/384/files#r1677583061.